### DISCUSSION TITLE
posts/test-three

### DISCUSSION THREAD

**Discussion by giscus** [teknologi-umum/blog#21] [General]
# posts/test-three

The post description. Might be something that would interest the reader in less than 100 words.

http://localhost:3000/posts/test-three?giscus=0dee617d80427f120e6af076HZ8qSekC7qrVtzlzwBPD%2FeUGKFSx%2BF%2BB88IVVUxAUUSh9xoL4rZKEQJY0BKua%2FKCc1kyt6%2FZ9hgNRaSmqZtKo%2FYt8qXTl9qR0QkUGAR6k8sAhDRFsbvUzoJbPlI%3D

**Comment by aldy505** [comment]
Hello world!

**Comment by elianiva** [comment]
`ハロ、世界！ != tidak halo dunia!`

**Reply by elianiva** [reply]
asik, bisa

**Comment by herlangga72** [comment]
problematikanya sama ya sama shiki? wrap textnya bisa nggak??
